faunadb/client_logger.py

```diff
@@ -0,0 +1,46 @@
+from ._json import to_json
+
+
+# Function suitable to be passed
+def logger(logger_func):
+  """
+  Function that can be the ``observer`` for a :any:`Client`.
+  :param logger_func:
+      Callback taking a string to be logged.
+  """
+  return lambda info: logger_func(show_request(info))
+
+
+def show_request(info):
+  """Translates a :any:`RequestResponseInfo` to a string suitable for logging."""
+  rq = info.request
```

Comment:
These are a tad short, can we use `req` and `resp` instead? The one-char difference is tripping me up a bit.